hopper/flash_attn_interface.py

```diff
@@ -210,8 +211,7 @@ def forward(
         ctx.deterministic = deterministic
         ctx.ndim = qkv.dim()
         ctx.sm_margin = sm_margin
-        # return out, softmax_lse
-        return out
+        return (out, softmax_lse) if return_softmax else out
```

Comment:
```suggestion
        return (out, softmax_lse, *rest) if return_softmax else out
```
Could also be done, but kept it simple - lmk if that should be changed